api/specs/webserver/v0/components/schemas/workbench.json

```diff
@@ -1,5 +1,6 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://simcore.io/api/specs/webserver/v0/components/schemas/workbench.json",
```

Comment:
do you still use this I thought it was a bad idea to split from project-v0.0.1.json?